src/inmanta/server/server.py

```diff
@@ -67,51 +63,14 @@ def get_dependencies(self) -> List[str]:
     def get_depended_by(self) -> List[str]:
         return [SLICE_TRANSPORT]
 
-    def define_features(self) -> List[Feature[object]]:
-        return [dashboard_feature]
-
     async def prestart(self, server: protocol.Server) -> None:
         self._server = server
         self._server_storage: Dict[str, str] = self.check_storage()
         self.compiler: "CompilerService" = cast("CompilerService", server.get_slice(SLICE_COMPILER))
-
-        self.setup_dashboard()
-
-    def setup_dashboard(self) -> None:
-        """
-        If configured, set up tornado to serve the dashboard
-        """
-        if not opt.dash_enable.get() or not self.feature_manager.enabled(dashboard_feature):
-            return
-
-        dashboard_path = opt.dash_path.get()
-        if dashboard_path is None:
-            LOGGER.warning("The dashboard is enabled in the configuration but its path is not configured.")
```

Comment:
We should also remove all dashboard-related config options from the server.

Reply:
I think this is done now?